src/MultiSelect/MultiSelect.less

```diff
@@ -0,0 +1,142 @@
+@import (reference) "../less/utilities";
+
+.MultiSelect--container {
+  font-family: "Proxima Nova";
+  position: relative;
+}
+
+.MultiSelect--label {
+  .padding--bottom--2xs();
+  color: @neutral_medium_gray;
+  .text--small();
+
+  /*
+    for a11y purposes
+    https://a11y-guidelines.orange.com/en/web/components-examples/accessible-hiding/#
+  */
+  &--hidden {
+    /* stylelint-disable */
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
+    white-space: nowrap; /* added line */
+    border: 0;
+    /* stylelint-enable */
+  }
+}
+
+.MultiSelect--selectContainer {
+  .text--medium();
+  position: relative;
+  width: 100%;
+  .inlineFlexbox();
+  .items--center();
+  .border--m(@neutral_silver);
+  .borderRadius--l();
+  background-color: @neutral_white;
+  .padding--left--xs();
+  .margin--y--3xs();
+  box-sizing: border-box;
+  transition: box-shadow @timingQuickly ease-out;
+
+  &--focused {
+    box-shadow: 0 0 0 @size_2xs ~"@{primary_blue_tint_2}80";
+
+    .border--m(@primary_blue);
+  }
+
+  /* stylelint-disable-next-line no-descending-specificity */
+  &:hover {
+    .border--m(@primary_blue);
+  }
+}
+
+.MultiSelect--selectedItemsContainer {
+  .flexbox();
+  .flex--wrap();
+  .items--center();
+  width: 100%;
+  .self--stretch();
+}
+
+.MultiSelect--selectedItemContainer {
+  .flexbox();
+  .margin--y--3xs();
+  .margin--right--2xs();
+}
+
+.MultiSelect--selectedItemButton {
+  .margin--left--2xs();
+  cursor: pointer;
+}
+
+.MultiSelect--input {
+  border: none;
+  outline: none;
+  .borderRadius--l();
+  width: 0%;
+  .flexGrow();
+
+  font-family: "Proxima Nova";
+  .text--medium();
+  line-height: @size_l;
+
+  &::placeholder {
+    //&::-webkit-input-placeholder, &::-moz-placeholder, &:-ms-input-placeholder {
```

Comment:
Are these prefixes not needed?

Reply:
I couldn't get them to work (seems like maybe related to this https://stackoverflow.com/a/41246875) so I commented them out. Looks it works when I separate the selectors out though